package.json

```diff
@@ -8,6 +8,14 @@
     "start": "next start"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-brands-svg-icons": "^5.15.2",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/react-fontawesome": "^0.1.14",
+    "@iconscout/unicons": "^3.0.6",
+    "@material-ui/core": "^4.11.3",
```

Comment:
What is this for

Reply:
the icons for Dribbble, Kaggle, Medium, google, etc are not available in material-ui/icons(already installed before)

so i installed fontawesome icons ....by installing @fortawesome package. It has all the free fontawesome icons.

The html-css-js version of this projects also used fontawesome. so I used also used it here